pkg/timestream/parsers.go

```diff
@@ -56,6 +56,30 @@ func getFieldBuilder(t *timestreamquery.Type) (*fieldBuilder, error) {
 				parser:    datumParserInt32,
 			}, nil
 
+		case timestreamquery.ScalarTypeIntervalDayToSecond:
+			return &fieldBuilder{
+				fieldType: data.FieldTypeNullableString,
```

Comment:
String does not seem like the right choice... maybe float64 for with the unit seconds?

Reply:
Okay I'll work on this. Is there a way to debug Go code better? Right now if I'm doing something wrong all I got was a 500 from front end.

Reply:
>  Is there a way to debug Go code better?

I find the easiest is to initiate things from VSCode in the backend directly.  In my working environment either move the value I care about first or comment out the other variables, then set breakpoints and debug tests from:

![image](https://user-images.githubusercontent.com/705951/108644506-1acb1300-7464-11eb-8b73-2b005943f840.png)


To debug go code where things are kicked off from the frontend is more complicated!  If the above method does not work for you, I can find some example vscode launcher scripts and add them to our repo

Reply:
Sounds easy enough. Let me try that.